flatpak/com.mitchellh.ghostty.yml

```diff
@@ -45,6 +45,7 @@ modules:
         --prefix /app
         --search-prefix /app
         --system $PWD/vendor/p
+      - glib-compile-schemas /app/share/glib-2.0/schemas
```

Comment:
For package managers this should be handled automatically, but the AppImage maintainer will probably need to run glib-compile-schemas when packaging.